### ISSUE TITLE
False positive NUnit1004 for test functions that receive array parameters

### ISSUE THREAD

**Issue by ssg** [nunit/nunit.analyzers#302]
The code below causes NUnit1004 to be fired incorrectly on v0.5.0, except with the case where array has only a single element:

```csharp
[TestCase(new byte[] { })]
[TestCase(new byte[] { 1 })] // doesn't trigger NUnit1004
[TestCase(new byte[] { 1, 2 })]
[TestCase(new byte[] { 1, 2, 3 })]
[TestCase(new byte[] { 1, 2, 3, 4 })]
public void Test(byte[] buffer)
{
  // doesn't matter
}
```

This doesn't happen on 0.4.0.

**Comment by Dreamescaper** [comment]
Thanks for reporting the issue, will take a look.

**Comment by mikkelbu** [comment]
Thanks for the bug report @ssg. I'll take a look at the PR now.

**Comment by ssg** [comment]
Wow this was fast, thank you everyone!
